pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -316,36 +362,43 @@ func (a *wlReconciler) reconcileGroup(ctx context.Context, group *wlGroup) error
 			}
 			// update the message
 			acs.Message = fmt.Sprintf("The workload got reservation on %q", reservingRemote)
+			// update the transition time
+			acs.LastTransitionTime = metav1.NewTime(time.Now())
+
 			wlPatch := workload.BaseSSAWorkload(group.local)
 			workload.SetAdmissionCheckState(&wlPatch.Status.AdmissionChecks, *acs)
 			err := a.client.Status().Patch(ctx, wlPatch, client.Apply, client.FieldOwner(ControllerName), client.ForceOwnership)
 			if err != nil {
-				return err
+				return reconcile.Result{}, err
 			}
 		}
-		// drop this if we want to create new remote workloads while holding a reservation
-		return nil
+		// drop this if we want to create new remote workloads while holding a reservation.
+		// check again the connection to the remote in half the keepReadyTimeout.
+		return reconcile.Result{RequeueAfter: a.keepReadyTimeout / 2}, nil
```

Comment:
Does this /2 mean we do two checks before giving up? If so, maybe we should make this two a named constant, and I would suggest having 3 checks, so /3?